init.rb

```diff
@@ -4,7 +4,7 @@
   name 'Redmine Issue Evm plugin'
   author 'Hajime Nakagama'
   description 'This is a plugin for Redmine. Earned value management using the ticket of redmine.'
-  version '3.5.3'
+  version '3.5.4'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.